flight/targets/brainre1/fw/Makefile

```diff
@@ -43,7 +43,7 @@ MODULES += Telemetry
 
 OPTMODULES += GPS
 OPTMODULES += PathPlanner
-OPTMODULES += AltitudeHold
+#OPTMODULES += AltitudeHold
```

Comment:
I left it because I plan to enable it again for the next release. But I can delete it as well